sources/packages/web/src/components/common/AssessmentAwardDetails.vue

```diff
@@ -180,33 +219,78 @@
             Final award
           </div>
           <!-- Final award table. -->
-          <award-table
-            :awardDetails="assessmentAwardData.finalAward"
-            :offeringIntensity="assessmentAwardData.offeringIntensity"
-            identifier="disbursementReceipt2"
-            v-if="showSecondFinalAward"
-          />
-          <div v-else>
+          <template v-if="showSecondFinalAward">
+            <award-table
+              :awardDetails="assessmentAwardData.finalAward"
+              :offeringIntensity="assessmentAwardData.offeringIntensity"
+              identifier="disbursementReceipt2"
+            />
+            <content-group-info v-if="allowFinalAwardExtendedInformation">
+              <div>
+                <span class="label-bold">Date eCert sent: </span>
+                <span>{{
+                  dateOnlyLongString(
+                    assessmentAwardData.estimatedAward
+                      .disbursement2DateSent as Date,
+                  )
+                }}</span>
+              </div>
+              <div>
+                <span class="label-bold">Certificate number: </span>
+                <span>{{
+                  assessmentAwardData.estimatedAward.disbursement2DocumentNumber
+                }}</span>
+              </div>
+            </content-group-info>
+            <!-- Part-time final awards are retrieved from e-Cert generated values and requires further explanation.  -->
+            <p
+              v-if="
```

Comment:
why cant we have the  check  assessmentAwardData.offeringIntensity === OfferingIntensity.partTime as a consumable and pass it here as isPartTime? and do the check

Reply:
By `consumable` do you mean `computed`?
If yes, I did not see a benefit for this case. Please let me know if it is a blocker.

Reply:
No difference in the reactivity in this case.
The computed would be better to not target the part-time/full-time for this case, IMO.
Either way at this stage I did not see it as a great improvement. Maybe if more logic is added on top of it.